### ISSUE TITLE
bestfit_and_interval not working in get_parameters_summary_statistics function

### ISSUE THREAD

**Issue by asofiafonso** [TommasoRonconi/galapy#4]
Hi Tommaso,

I am using Galapy and I am trying to get the age-best fit values for in-situ SFH. When using the `get_parameters_summary_statistic` function, I am getting an error that `self` is not defined. I tried to fix this by replacing the `self` with the `res` class object but I am now having issues with the `get_credible_interval` function. 

Can you help?

Best,
Ana

**Comment by TommasoRonconi** [comment]
Hi Ana,

thanks for the feedback. Can you give me more information to reproduce the error?
I have tried the function with the results from a fitting run of mine and did not find any problem:

```python
from galapy.sampling.Results import load_results
res = load_results('/path/to/myfile_results.galapy.hdf5')

from galapy.analysis.funcs import get_parameters_summary_statistics
get_parameters_summary_statistics(res)
```

Let me know.

Best,
Tommi

**Comment by asofiafonso** [comment]
Hi again,

Thanks a lot for your quick answer.

The default mode is working. The issue happens when I define the `stat_type = 'bestfit_and_interval'`.

Can you have a look?

Thanks!

Cheers,
Ana

**Comment by TommasoRonconi** [comment]
Ciao, 
thanks for the clarification.

I managed to reproduce the error and marked this issue as a bug report.
The solution is on its way, as soon as I upload it I will close this issue.

Best,
Tommi

**Comment by TommasoRonconi** [comment]
Solved in `v0.5.3`
